infernyx/rules.py

```diff
@@ -35,6 +37,9 @@
                     'rm', 'ro', 'ru', 'si', 'sk', 'sl', 'son', 'sq', 'sr', 'sv-se', 'sw', 'ta', 'te', 'th',
                     'tr', 'uk', 'ur', 'vi', 'xh', 'zh-cn', 'zh-tw', 'zu'}
 
+AS_HLL_MAP_KEYS = ['date', 'release_channel', 'version', 'country_code']
```

Comment:
remove these constants and just put them in the rule as key_parts and value_parts